q2_emperor/plugin_setup.py

```diff
@@ -18,16 +18,21 @@
 
 
 def plot(output_dir: str, sample_metadata: qiime.Metadata,
-         pcoa: skbio.OrdinationResults) -> None:
+         pcoa: skbio.OrdinationResults, custom_axis: str=None) -> None:
 
     mf = sample_metadata.to_dataframe()
 
+    if custom_axis not in mf.columns and custom_axis:
+        raise KeyError("The custom_axis category %s is not present in the "
+                       "metadata." % custom_axis)
```

Comment:
We should have error checking here to confirm that there are no nan's in the column, as those will cause Emperor to hang on loading.